protocol/communities/community_changes.go

```diff
@@ -84,6 +88,22 @@ func (c *CommunityChanges) HasMemberLeft(identity string) bool {
 	return ok
 }
 
+func (c *CommunityChanges) HasMemberBanned(identity string) bool {
+	if len(c.MembersBanned) == 0 {
+		return false
+	}
+	_, ok := c.MembersBanned[identity]
```

Comment:
na: If members are stored in `map[string]bool` maybe it's better to check result && ok

Reply:
go does not have a set, so in this case value is always true, no need to check if, I want to see if key present in the map only